packages/core/src/browser/browser.ts

```diff
@@ -11,7 +11,7 @@
 
 const userAgent = typeof navigator !== 'undefined' ? navigator.userAgent : "";
 
-export const isIE = (userAgent.indexOf('Trident') >= 0);
+export const isIE: boolean = (userAgent.indexOf('Trident') >= 0);
```

Comment:
Why adding the type here explicitly?

Reply:
Left over when I was editing files with Theia + Textmate to see type annotations, I will remove it.